openapi_python_client/parser/properties/model_property.py

```diff
@@ -27,6 +27,7 @@ class ModelProperty(Property):
     additional_properties: Union[bool, Property]
 
     template: ClassVar[str] = "model_property.pyi"
+    json_is_dict: ClassVar[bool] = True
```

Comment:
You should add this classvar to the parent class to be consistent. It's not erroring now because Jinja is permissive, but if you were to do `property.json_is_dict` outside of a template, it would error for non-model properties.

```python
json_is_dict: ClassVar[bool] = False
```